main.sh

```diff
@@ -19,8 +19,8 @@ sort -n -k 10,10 hg19.coding_subset.bed > hg19.coding_subset.sorted.bed
 awk '$10==3' hg19.coding_subset.sorted.bed > hg19.coding_subset.3exon.sorted.bed
 
 ## Task4: extract transcripts with 3 exons and chr17 from the sorted file from task2. Name the new file "hg19.coding_subset.3exon.chr17.sorted.bed" 
-
+awk '/chr17/' hg19.coding_subset.sorted.bed | awk '$10==3' > hg19.coding_subset.3exon.chr17.sorted.bed
```

Comment:
This is working but actually you can have a complete "if statement" with multiple conditions to check for the value in multiple columns.